build.sh

```diff
@@ -165,12 +159,38 @@ OSARGS="-s ${IOS_MIN_SDK_VERSION} -t ${TVOS_MIN_SDK_VERSION} -i ${MACOS_X86_64_V
 
 ## Welcome
 echo -e "${bold}Build-OpenSSL-cURL${dim}"
-if [ "$catalyst" != "" ]; then
-	echo "This script builds OpenSSL, nghttp2 and libcurl for MacOS, Catalyst, iOS and tvOS devices."
+echo
+echo -e " - OpenSSL version: ${subbold}${OPENSSL}${dim}"
+echo -e " - cURL version:    ${subbold}${LIBCURL}${dim}"
+if [ "$buildnghttp2" == "" ]; then
+	echo ""
+	echo -n "This script builds OpenSSL and libcurl for "
 else
-	echo "This script builds OpenSSL, nghttp2 and libcurl for MacOS, iOS and tvOS devices."
+	echo -e " - nghttp2 version: ${subbold}${NGHTTP2}${dim}"
+	echo ""
+	echo -n "This script builds OpenSSL, nghttp2 and libcurl for "
+fi
+case "$BUILDFOR" in
+	ios) echo -e "${archbold}iOS${dim}" ;;
+	tvos) echo -e "${archbold}tvOS${dim}" ;;
+	macos) echo -e "${archbold}macOS${dim}" ;;
+	*) 
+	    if [ "$catalyst" != "" ]; then
+			echo -e "${archbold}macOS${dim}, ${archbold}catalyst${dim}, ${archbold}iOS${dim} and ${archbold}tvOS${dim}"
+		else
+			echo -e "${archbold}macOS${dim}, ${archbold}iOS${dim} and ${archbold}tvOS${dim}"
+		fi
+		;;
+esac
+
+## Ask for confirmation to continue
+if [ "$autobuild" == "" ]; then
+	read -p "Continue (Y/n)? " choice
+	case "$choice" in 
+		n|N ) echo "Exiting"; exit 1 ;;
+		* ) echo "Continuing..."; echo "" ;;
+	esac
```

Comment:
Added build details (what is being built) and a user confirmation prompt before proceeding. This will help with those accidental builds that step on your working build, or if you didn't specify the right options/versions. Additionally, added a build `-y` option to auto-yes this prompt: 

```bash
./build.sh -y
```